src/mame/trs/coco12.cpp

```diff
@@ -708,26 +866,28 @@ ROM_END
 
 #define rom_cocoh rom_coco
 #define rom_cocoeh rom_cocoe
+#define rom_deluxecoco rom_coco2b
```

Comment:
The proper way to designate that we don't have the real ROM is to use:
`ROM_LOAD(..., ..., ..., NO_DUMP)`

Reply:
marked deluxecoco's ROMs as NO_DUMP